File: TiaCodegen/Commands/Comparisons/Le.cs
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands.Comparisons
{
    public class Le : CompareOperator // <=
    {
        public Le(params IOperationOrSignal[] operationOrSignals)
            : base(operationOrSignals)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/Comparisons/Le.ts
import { IOperationOrSignal } from '../../Interfaces/IOperationOrSignal';
import { CompareOperator } from './CompareOperator';

export class Le extends CompareOperator {
    constructor(...operationOrSignals: IOperationOrSignal[]) {
        super(...operationOrSignals);
    }
}
